iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/shareaza/LibraryHistory.java

```diff
@@ -49,8 +49,8 @@ public void read(MFCParser ar, int version) throws IOException {
             lastSeededTorrentPath = ar.readString();
             if (lastSeededTorrentPath.length() > 0) {
                 lastSeededTorrentName = ar.readString();
-                lastSeededTorrentTime = Util.formatDatetime(Util.convertToEpoch(ar.readUInt()));
```

Comment:
Hi @tc-wleite, I just tested this on my small Shareaza test set, seems no regressions were found. But I'm curious about this `Util.convertToEpoch()` removal, it seems to change this returned value. Was it incorrect before? @fmpfeifer

Reply:
I took this from the changes in latest version of Python parser. 
In the file that I sent privately to you,  before this change the year was 1600 (or something like that), and now seems a more plausible date.

Reply:
Great! I'll take a look at this date on my files tomorrow, thank you!